lib/utils.js

```diff
@@ -53,20 +53,25 @@ function appender(xs) {
     return xs
   }
 }
-
+function padStart(number, expectedLength) {
+  const strNumber = String(Math.abs(number)).padStart(expectedLength, '0')
+  return (number < 0) ? `-${strNumber}` : strNumber
+}
 // Custom Logger
 function log(message, level) {
   const dateObject = new Date()
-  const padStart = (number, expectedLength) => {
-    const strNumber = String(Math.abs(number)).padStart(expectedLength, '0')
-    return (number < 0) ? `-${strNumber}` : strNumber
-  }
-  const date = `${dateObject.getFullYear()}-${padStart(dateObject.getMonth() + 1, 2)}-${padStart(dateObject.getDate(), 2)}`
-  const time = `${padStart(dateObject.getHours(), 2)}:${padStart(dateObject.getMinutes(), 2)}:${padStart(dateObject.getSeconds(), 2)}`
+  const date = `${dateObject.getFullYear()}-${
+    padStart(dateObject.getMonth() + 1, 2)
+  }-${padStart(dateObject.getDate(), 2)}`
+  const time = `${padStart(dateObject.getHours(), 2)}:${
+    padStart(dateObject.getMinutes(), 2)
+  }:${padStart(dateObject.getSeconds(), 2)}`
```

Comment:
I'd just add `luxon` as a dependency and use `.toFormat` https://moment.github.io/luxon/#/formatting?id=toformat. It'd be a lot easier to read and understand.

Reply:
It's legacy code and this is just a format change. I could just add that. I am not trying to put too much into this as this repo will eventually be deprecated

Reply:
I am not blocking anything, but I think it is a good idea to consider the duration of "eventually deprecated", how long that deprecation will last, and if the tool will be maintained during that deprecation period. In my opinion, I think easier to maintain code would be better.

Reply:
I'm afraid to say that this is going to be around a while.  There are customers that will never upgrade, so we should find that balance between changing too much and doing things that'll help us maintain it.  AFAIK, we'd only be bumping node and patching critical dependency updates when they come up.